WalletLibrary/WalletLibrary/Requests/OpenId/OpenIdPresentationRequest.swift

```diff
@@ -65,14 +72,52 @@ class OpenIdPresentationRequest: VerifiedIdPresentationRequest
     }
     
     /// Completes the request and returns a Result object containing void if successful, and an error if not successful.
-    func complete() async -> VerifiedIdResult<Void> {
+    func complete() async -> VerifiedIdResult<Void> 
+    {
+        /// TODO: remove feature flag in next PR.
+        if configuration.isPreviewFeatureFlagSupported(PreviewFeatureFlags.PresentationExchangeSerializationSupport)
+        {
+            return await completeWithProcessorExtensions()
+        }
+        
         return await VerifiedIdResult<Void>.getResult {
             var response = try PresentationResponseContainer(rawRequest: self.rawRequest)
             try response.add(requirement: self.requirement)
             try await self.responder.send(response: response)
         }
     }
     
+    private func completeWithProcessorExtensions() async -> VerifiedIdResult<Void>
+    {
+        return await VerifiedIdResult<Void>.getResult {
+            
+            /// Only support Verifiable Credential Serializer for now.
```

Comment:
If we only support these (as protocols can assert), why are they injected using protocols?

Reply:
its more of a TODO to support that as soon as I can.

Reply:
Before we publish a new version.